sys/net/network_layer/fib/universal_address.c

```diff
@@ -197,6 +197,35 @@ int universal_address_compare(universal_address_container_t *entry,
     return ret;
 }
 
+int universal_address_compare_prefix(universal_address_container_t *entry,
+                              uint8_t *prefix, size_t prefix_size)
```

Comment:
the `prefix_size` refers to the full length of the address type used in `prefix`, right?
In case of a prefix for ipv6 addresses `2001:db8:3c4d::/48`, I would  still use sizeof(ng_ipv6_addr) instead of 48? Therefore, a "prefix address" MUST have trailing zeros? Can you add to the doc, that trailing zeros are a MUST for prefix checks, which are not ::/128 (in case of ipv6)?